### PR TITLE
Add Distribute

### PR DESCRIPTION
**Pull request by sebweb3r**
More words

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -9602,6 +9602,11 @@ distingusih->distinguish
 distinquish->distinguish
 distinquishable->distinguishable
 distintions->distinctions
+distirbute->distribute
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:9605]: Add distributes variant

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -22031,6 +22036,9 @@ redircet->redirect
 redirectd->redirected
 redirectrion->redirection
 redisign->redesign
+redistirbute->redistribute
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:22039]: Again add the plural.

**Review by peternewman — CHANGES_REQUESTED** [review]
A few comments

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (sebweb3r) [reply codespell_lib/data/dictionary.txt:9605]: added.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (sebweb3r) [reply codespell_lib/data/dictionary.txt:22040]: added.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -9602,6 +9602,12 @@ distingusih->distinguish
 distinquish->distinguish
 distinquishable->distinguishable
 distintions->distinctions
+distirbute->distribute
+distirbuted->distributed
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:9606]: I guess "disturbed" is probably too-different to suggest here?

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -22031,6 +22037,10 @@ redircet->redirect
 redirectd->redirected
 redirectrion->redirection
 redisign->redesign
+redistirbute->redistribute
+redistirbuted->redistributed
+redistirbutes->redistributes
+redistirbuting->redistributing
```
Comment (lurch) [inline codespell_lib/data/dictionary.txt:22043]: redistribution ?

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (sebweb3r) [reply codespell_lib/data/dictionary.txt:9606]: I agree

Comment (peternewman) [reply codespell_lib/data/dictionary.txt:9606]: Yes, it is.

**Review by peternewman — APPROVED** [review]

**Pull request merged**